docs/user-guides/f-map-styles/1-base-map-styles.md

```diff
@@ -1,10 +1,19 @@
 # Base Map Styles
 
 Open the __Base Map panel__ to modify the base map color scheme, imagery, and labeling.
-![base map panel](https://d1a3f4spazzrp4.cloudfront.net/kepler.gl/documentation/image9.png "base map panel")
+![base map panel](https://d1a3f4spazzrp4.cloudfront.net/kepler.gl/documentation/f-map_style_panel.png "base map panel")
```

Comment:
you may want to add a new line in between these two otherwise the image won't wrap to next line in visual mode